resources/lib/playerinfo.py

```diff
@@ -189,17 +189,15 @@ def push_upnext(self):
 
     def update_position(self):
         ''' Update the player position, when possible '''
-        try:
-            self.last_pos = self.getTime()
-        except RuntimeError:
-            pass
+        if not self.isPlaying():
+            return
+        self.last_pos = self.getTime()
```

Comment:
This is not correct. It may be playing when you check, and it may have stopped when you get the position.

The probability that this happens is very small, but it can still happen. Unfortunately, exception handling in this case is the only real option. (and yes, almost every other add-on I have seen are doing this wrong as well)

Reply:
Okay, I'll close this.